18/04/21 (13:19:56) | ERROR: [Errno 2] No such file or directory: '/home/corey/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/local/bin/bpytop", line 1319, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/corey/.config/bpytop/themes'
19/04/21 (09:03:24) | ERROR: [Errno 2] No such file or directory: '/home/corey/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/local/bin/bpytop", line 1319, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/corey/.config/bpytop/themes'
